.github/workflows/shiftleft.yml

```
@@ -0,0 +1,62 @@
+  NextGen-Static-Analysis:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Java JDK v8
+      uses: actions/setup-java@v3
+      with:
+        distribution: zulu
+        java-version: 8
+
+    - name: Download ShiftLeft CLI
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:45: Double quote to prevent globbing and word splitting

(shellcheck)